com.vivaldi.Vivaldi.yaml

```diff
@@ -0,0 +1,136 @@
+app-id: com.vivaldi.Vivaldi
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '22.08'
+command: vivaldi
+separate-locales: false
+finish-args:
+  - --device=all # Webcams and others
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf # For GNOME proxy resolution
+  - --env=GIO_EXTRA_MODULES=/app/lib/gio/modules # For GNOME proxy resolution
+  - --env=GSETTINGS_BACKEND=dconf # For GNOME proxy resolution
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --filesystem=home/.local/share/applications:create # Allow for PWAs
+  - --filesystem=home/.local/share/icons:create # Allow for PWAs
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-download # Default download directory
+  - --filesystem=xdg-run/dconf # For GNOME proxy resolution
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=~/.config/dconf:ro # For GNOME proxy resolution
+  - --filesystem=~/.config/google-chrome:ro # Import bookmarks and other data from a classic Chrome install
+  - --filesystem=~/.config/kioslaverc # For KDE proxy resolution (KDE5 only)
+  - --filesystem=~/.local/lib:create # Extra FFMpeg saved here
+  - --filesystem=~/.mozilla/firefox:ro # Import bookmarks and other data from a classic FF install
```

Comment:
all these filesystem access really defeat having that proprietary blob in a sandbox.

Reply:
Fair but most of them are the same as the other Chromium browsers on flathub already. I will drop the Firefox and Chrome read only ones and let onboarding suffer and see if I can tweak the `--filesystem=~/.local/lib:create` one